### PR TITLE
Fix setup for conda

### LINKED ISSUE #10: Sagemath and binder do not work together (neither docker nor conda)
I made several tries to work with sagemath and mybinder with no success:

- Docker with sagemathinc/cocalc:latest used to work, it does not any more for me.
- Tried also with cocalc-lite, same thing.
In both cases basically the same error as in [this post](https://discourse.jupyter.org/t/binder-and-sagemath/11059/2?u=enriqueartal)
I tried the conda approach both the tests and one of my repos (not explained in the readme, just disable docker file and copy environment.yml. In this case binder session is opened but sage kernel is not connected. Running `sage` in the terminal gives the error in [this other post](https://discourse.jupyter.org/t/binder-and-sagemath/11059/9?u=enriqueartal).

### PR DESCRIPTION
**Pull request by saraedum**
by preferring conda-forge packages over defaults.

Fixes #10.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,12 +1,19 @@
 name: test-binder-sage
 
 channels:
-  - defaults
```
Comment (saraedum) [inline environment.yml:4]: The main thing of this PR is that `conda-forge` should always come before `defaults`.

**Comment by nthiery** [comment]
Ah ah, good catch.

Do we even need the `default` channel? I don't even remember why it's there in the first place.

**Comment by nthiery** [comment]
Thank you for the fix, and for the useful information for people wanting to build
their own binder.

**Comment by nthiery** [comment]
Incidentally: I see you added jupyterlab as dependency. Shall we update the binder link
to use jupyterlab by default?

**Comment by nthiery** [comment]
This is now #13

**Pull request merged**